src/Bicep.Core/Semantics/Namespaces/SystemNamespaceType.cs

```diff
@@ -149,6 +149,22 @@ static IEnumerable<FunctionOverload> GetAlwaysPermittedOverloads()
                     .WithRequiredParameter("network", LanguageConstants.String, "String in CIDR notation containing an IP address range to be converted.")
                     .Build();
 
+                yield return new FunctionOverloadBuilder("buildUri")
+                    .WithReturnType(LanguageConstants.String)
+                    .WithGenericDescription("Constructs a URI from specified components including scheme, host, port, path, and query.")
+                    .WithRequiredParameter("scheme", LanguageConstants.String, "The URI scheme (e.g., http, https).")
```

Comment:
`buildUri()` only has one parameter that is an object with several properties. This declares multiple parameters.